src/main/java/com/iridium/iridiumskyblock/database/User.java

```diff
@@ -181,4 +181,11 @@ public void setCurrentIslandVisiting(Island currentIslandVisiting) {
     public void setTeleportingTask(BukkitTask teleportingTask) {
         this.teleportingTask = teleportingTask;
     }
+
+    //moved method
+    public User withIsland(Island island) {
```

Comment:
I dont like this being here, this is only needed for unit tests, so should stay coupled to the unit tests UserBuilder